data/transport missions.txt

```diff
@@ -2893,3 +2893,59 @@ mission "Pirate Blockade"
 			`	You and the man, who does not give his name to you in order to "keep us both safe," wait for the woman to arrive in her own ship. Hours pass, but there is no sign of her. The man tries to contact her, but he is unable to reach her. "You can leave now," the man says to you. "I don't think she'll be coming."`
+
+
+
+mission "Capture Smuggler"
+	name "Capture rogue smuggler"
+	minor
+	description `A smuggler absconded with his illegal cargo near <system>. The unknown person he stole it from wants it back. Recover the cargo from the <npc> and return both it and the smuggler to <origin> for <payment>.`
+	to offer
+		random < 15
+		"combat rating" > 2000
+		"reputation: Pirate" > 10
+	passengers 1
+	source
+		government "Pirate"
+	on offer
+		conversation
+			`As you step into a dive bar for a quick drink, the largest man you've ever seen stands up and blocks your path. He's well over two meters tall, and his muscles seem to ripple with barely restrained violence when he moves. "<first> <last>," he says in a thick accent. "My employer hears you do certain jobs, take money, do not ask questions. He hears you are reliable. A man, a smuggler, he steals cargo from my employer. My employer very much wants cargo returned, and this man as well."`
+			choice
+				`	"I'm sorry, sir, but I'm not taking new jobs today. Please give your employer my most gracious thanks for this kind offer."`
+				`	"Of course. I won't say a word."`
+					goto accept
+				`	"Get out of my way. I don't take jobs from any schlub who bothers me when I need a drink."`
+					goto mistake
+
+			`	The man looks down at you with no change in expression. "My employer, he will not be happy. I will tell him, '<first> <last> is not so reliable.' I hope next time, you are reliable." As the man speaks, you notice that his teeth are filed to vicious points. Every patron in the bar stares at you in shock as you sit down for a badly-needed drink.`
+				decline
+
+			label accept
+			`	The man looks down at you with no change in expression. "This is good. To be reliable, it is a great thing on <origin>." As the man speaks, you notice that his teeth are filed to vicious points. He holds out a round, flat object about the size of your palm. It has a recessed button in the center of one side. "This is gift for this smuggler. Press button, it becomes surprise. Make sure surprise is not on you." He hands you the object and a datachip containing details of the job. As the man leaves, you see every patron in the bar staring warily at his back.`
+				accept
+
+			label mistake
+			`	The man looks down at you with no change in expression. "This is great shame. I will tell my employer, '<first> <last> was not so reliable. I had to make example.'" The man grabs your neck with one monstrous hand. The last thing you see as your vision fades to black is a mouth full of pointed teeth.`
+				die
```

Comment:
Maybe `flee` instead of `die`? Not to mention that this is a pretty graphic death scene.

Reply:
Already merged. You can always send in a new PR though, it's open source after all.

Reply:
The `die` end state honestly isn't used enough, but the best place that it could probably be used more is in pirate missions like this where disrespecting the wrong person can wind up getting you killed.

Reply:
Fair enough about `die`. Anyone else thinks the scene might be too graphic? I don't think there's any other place in the game where death is _this_ graphic, it's just ships exploding or "the officer ducked and someone shot you".

Reply:
The FW alpha missions come to mind.

>`While Nguyen holds the creature still, one of the Navy troops puts a gun to its head and pulls the trigger. Not even an Alpha can survive that; the creature goes limp.`
Ref:
https://github.com/endless-sky/endless-sky/blob/b3ea610de5b2a9c3a9c1014b78e944aa21e2b7ec/data/free%20worlds%20middle.txt#L1134